HelperFunctions/PeakFitting/voigtN.m

```diff
@@ -0,0 +1,37 @@
+function [ fit_type ] = voigtN(n , bg)
+    %VOIGTN Creates fittype of n(>1) Pseudo-Voigt approximation functions with one optional y-offset
+    %   Default is to include offset
+    %   https://en.wikipedia.org/wiki/Voigt_profile
+
+    assert(n==round(n), 'n must be an integer.')
+    assert(isnumeric(n)&&n>0, 'n must be greater than 0.')
+    % bg specifies including background offset
+    if nargin < 2
+        bg = true;
+    end
+    
+    % a = amplitude,
+    % b = center,
+    % cl = Lorentz width,
+    % cg = Gauss width,
+    % d = y-offset.
+    subEQ = @(n)sprintf('voigt(x, a%i, b%i, cl%i, cg%i)',n,n,n,n);
```

Comment:
This seems like an old version. Didn't Eric move to fitting eta and f instead of fl and fg